### ISSUE TITLE
Retransmitter not created while offer is not in expected format with RTX/9000

### ISSUE THREAD

**Issue by zhiyua-git** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#188]
When we receive offer below, it is not containing RTX/9000 we are looking for in parser, so the ssrc of RTX was not set up properly and retransmitter become null while there is NACK setup in offer. Need to parse it properly to get retransmitter up and working against NACK:
```
v=0
o=- 3790131452 3790131452 IN IP4 0.0.0.0
s=a 2 z
c=IN IP4 0.0.0.0
t=0 0
a=group:BUNDLE audio0 video0
m=audio 1 UDP/TLS/RTP/SAVPF 96 0
a=candidate:1 1 UDP 2013266431 3.82.233.177 53989 typ host
a=candidate:3 1 TCP 1010827519 3.82.233.177 46277 typ host tcptype passive
a=candidate:2 1 TCP 1015021823 3.82.233.177 9 typ host tcptype active
a=candidate:2 2 TCP 1015021822 3.82.233.177 9 typ host tcptype active
a=candidate:3 2 TCP 1010827518 3.82.233.177 60543 typ host tcptype passive
a=candidate:1 2 UDP 2013266430 3.82.233.177 64256 typ host
a=setup:actpass
a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=rtpmap:96 opus/48000/2
a=rtcp:9 IN IP4 0.0.0.0
a=rtcp-mux
a=sendrecv
a=mid:audio0
a=ssrc:1186017576 cname:user3132994977@host-56e22ea7
a=ice-ufrag:U9gx
a=ice-pwd:110iT/6/nHRPYE0Lo6vUyr
a=fingerprint:sha-256 D7:3F:C1:F6:C5:6B:FF:D9:80:17:5A:5F:47:3A:EA:60:E2:80:93:4E:5C:87:52:1D:56:E9:15:81:67:D9:DB:AB
m=video 1 UDP/TLS/RTP/SAVPF 99
a=candidate:1 1 UDP 2013266431 3.82.233.177 53989 typ host
a=candidate:3 1 TCP 1010827519 3.82.233.177 46277 typ host tcptype passive
a=candidate:2 1 TCP 1015021823 3.82.233.177 9 typ host tcptype active
a=candidate:2 2 TCP 1015021822 3.82.233.177 9 typ host tcptype active
a=candidate:1 2 UDP 2013266430 3.82.233.177 64256 typ host
a=candidate:3 2 TCP 1010827518 3.82.233.177 60543 typ host tcptype passive
b=AS:2500
a=setup:actpass
a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=rtpmap:99 H264/90000
a=rtcp:9 IN IP4 0.0.0.0
a=rtcp-mux
a=sendrecv
a=mid:video0
a=rtcp-fb:99 nack
a=rtcp-fb:99 nack pli
a=rtcp-fb:99 ccm fir
a=ssrc:2565345525 cname:user3132994977@host-56e22ea7
a=ice-ufrag:U9gx
a=ice-pwd:110iT/6/nHRPYE0Lo6vUyr
a=fingerprint:sha-256 D7:3F:C1:F6:C5:6B:FF:D9:80:17:5A:5F:47:3A:EA:60:E2:80:93:4E:5C:87:52:1D:56:E9:15:81:67:D9:DB:AB
```

**Comment by MushMal** [comment]
Perfect! Aside from the hardening for NULL check we should be able to fix this and create a specific unit test to track. @Sean-Der could you please comment on what the desired behavior should be in this case or similar case.

**Comment by Sean-Der** [comment]
Fixed by 6fd363dbd854a510a552ae55bbb61f6ead9eed46

If there is a distinct PayloadType for RTX we just re-send the original packet now.
